src/main/toFile.csv

```diff
@@ -0,0 +1,2 @@
+banana,152
+apple,90
```

Comment:
Add space in the end of file

Reply:
locally it is present
![image](https://user-images.githubusercontent.com/115244741/224730108-5def077f-05aa-410c-9def-6c678edc122f.png)